src/app/layout/navbar/navbar.component.html

```diff
@@ -1,5 +1,5 @@
 <header>
-  <nav class="transparent">
+  <nav class="white">
```

Comment:
Transparent style is used when page is at the top, so it start changing when you scroll down.